.gitignore

```diff
@@ -10,4 +10,6 @@ obj/
 /WebAssets/dist
 /WebAssets/coverage
 /Owin.Demo/wwwroot
-/AspNetCore.Demo/wwwroot
\ No newline at end of file
+/AspNetCore.Demo/wwwroot
+/AspNetCore.Demo/dist
+.idea/
```

Comment:
I would put this on top next to `.vs/`, but I don't mind adjusting it myself as well.